vipps-login-api-faq.md

```diff
@@ -156,6 +157,9 @@ electronic ID, e.g  BankID.
 
 Merchants need to apply for access to NNIN separately by sending an email to AccessUserInfo@vipps.no. In the email you should specify merchant name and organization number and name of sales unit from [VippsPortalen](https://portal.vipps.no). You also need to provide information on how you plan to use the NNIN, the legal requirement and/or the reason why you need to use NNIN to achieve required user identification.
```

Comment:
We could also here add: Access to NNIN is a payed service.

Reply:
Added.